### PR TITLE
fix: capture dismiss when selecting field in chart config

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: #2845 

### Description:

- removed redundant code from `packages/frontend/src/components/ChartConfigPanel/index.tsx` 
- added `captureDismiss` in the field autocomplete component so that it prevents the chart config from closing
- had to update all blueprintjs so the property from point above would work 
- had to fix date input default values

capture dismiss

### CODE REVIEW

File: packages/frontend/src/components/ChartConfigPanel/index.tsx [hunk 1]
```diff
@@ -1,23 +1,19 @@
...
 export const ChartConfigPanel: React.FC = () => {
     const { chartType, plotData } = useVisualizationContext();
     const disabled = !plotData || chartType === ChartType.TABLE;
 
-    const [isOpen, setIsOpen] = useState(false);
     return (
         <Popover2
             content={<ChartConfigTabs />}
             interactionKind="click"
-            isOpen={isOpen}
```
Comment (rephus) [inline packages/frontend/src/components/ChartConfigPanel/index.tsx:17]: does it mean it is "always open" now ?

File: packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx [hunk 2]
```diff
@@ -108,7 +108,7 @@ const DateFilterInputs: FC<FilterInputsProps<DateFilterRule>> = (props) => {
                         parseDate={(value) =>
                             moment(value, `YYYY-MM-DD, HH:mm:ss:SSS`).toDate()
                         }
-                        defaultValue={new Date()}
+                        defaultValue={new Date().toString()}
```
Comment (rephus) [inline packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx:111]: is it better to use `toISOString` or a shorter format ? I don't even know why you need to do this

**Review by rephus — APPROVED** [review]
Works as expected, so :sheep: it

File: packages/frontend/src/components/ChartConfigPanel/index.tsx (hunk 1 above)
Comment (ZeRego) [reply packages/frontend/src/components/ChartConfigPanel/index.tsx:17]: No, it means we don't control the popover open state, it knows when to open and close.

File: packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx (hunk 2 above)
Comment (ZeRego) [reply packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx:111]: blueprint changed the `defaultValue` typing from Date to string. I don't think it matter much how the string is formatted.

**Pull request merged**